lrtable/src/lib/statetable.rs

```diff
@@ -947,6 +947,44 @@ mod test {
                    Action::Reduce(grm.rule_to_prods(grm.rule_idx("Expr").unwrap())[0]));
     }
 
+    #[test]
+    fn conflicts() {
+        let grm = YaccGrammar::new(
+            YaccKind::Original,
+            &"
+%start A
+%%
+A : 'a' 'b' | B 'b';
+B : 'a' | C;
+C : 'a';
+          "
+        )
+        .unwrap();
+        let sg = pager_stategraph(&grm);
+        let st = StateTable::new(&grm, &sg).unwrap();
+        let conflicts = st.conflicts().unwrap();
+        assert_eq!(conflicts.sr_len(), 1);
+        assert_eq!(conflicts.rr_len(), 1);
+        let mut itersr = conflicts.sr_conflicts();
+        assert_eq!(
+            itersr.next().unwrap(),
+            &(
+                grm.token_idx("b").unwrap(),
+                grm.rule_to_prods(grm.rule_idx("B").unwrap())[0],
+                StIdx::from(2)
+            )
+        );
+        let mut iterrr = conflicts.rr_conflicts();
+        assert_eq!(
```

Comment:
Would it be easier to say `assert_eq!(conflicts.rr_conflicts().next().unwrap(), ...);`?